fullstack-cert/css-projects/pricing-plans-layout/styles.css

```diff
@@ -0,0 +1,129 @@
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+h1 {
+  margin-bottom: 20px;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.pricing-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center; /* controls vertical spacing if there are multiple rows */
```

Comment:
All comments in this css files should be removed since this is a lab and not a workshop.